NextDom/nextdom-core
# fix issue 1477 : expression test

fix #1477

To test : 
in scenario page
- check expression test page
- variable modale
- globale view modale
